.travis.yml

```diff
@@ -2,7 +2,7 @@ language: node_js
 node_js:
   - 0.12
   - 4
-  - 5
+  - 5.5
```

Comment:
I removed 5.6 because of https://github.com/mcollina/mosca/issues/412

Is this no issue anymore?